app/(main)/league/[leagueId]/entry/[entryId]/week/Week.interface.ts

```diff
@@ -11,6 +23,18 @@ export interface IWeekParams {
   };
 }
 
+export interface IWeeklyPickChange {
+  data: IData;
+  entry: string;
+  league: string;
+  NFLTeams: INFLTeam[];
+  setUserPick: React.Dispatch<React.SetStateAction<string>>;
+  updateWeeklyPicks: ({}: IWeeklyPicks) => void;
+  user: IUser;
+  weeklyPicks: IWeeklyPicks;
+  week: string;
```

Comment:
Do we have an ENUM for the weeks?

Reply:
we do not. when i looked into it, it is just something that gets passed in as a string

Reply:
nit: We should look to adding it later. There's only week 1-18 available.